plugin/src/py/android_screenshot_tests/test_pull_screenshots.py

```diff
@@ -87,6 +89,15 @@ def _valid_src(self, src):
     def get_external_data_dir(self):
         return "/sdcard"
 
+    def _copy(self, src, dst):
+        try:
+            shutil.copytree(src, dst)
+        except OSError as exc:
+            if exc.errno == errno.ENOTDIR:
+                shutil.copy(src, dst)
+            else:
+                raise
+
```

Comment:
In order to properly support `test_run_id` in unit test we need to copy files recursively as `test-run-id`s adds folders as well